cev_eris.dme

```diff
@@ -407,6 +407,7 @@
 #include "code\datums\underwear\underwear.dm"
 #include "code\datums\uplink\ammunition.dm"
 #include "code\datums\uplink\announcements.dm"
+#include "code\datums\uplink\armor and apparel.dm"

```

Comment:
No spaces in file names please. It appears to work but we are dealing with BYOND here. Better safe than weird-ass unexpected BYOND bugs down the line.